lib/util/createHash.js

```diff
@@ -33,11 +33,17 @@ class BulkUpdateDecorator {
 		if(this.buffer.length > 0) {
 			this.hash.update(this.buffer);
 		}
-		return this.hash.digest(encoding);
+		var digestResult = this.hash.digest(encoding);
+		if(typeof digestResult !== "string")
```

Comment:
Wanted to note that
```
return(typeof digestResult !== "string") ?.....
```
Fails with `37:3  error  Expected space(s) after "return"  keyword-spacing` from eslint
and spaced return fails with prettier:
```
-		return (typeof digestResult === "string") ? digestResult : digestResult.toString();
+		return(typeof digestResult === "string") ? digestResult : digestResult.toString();
```
Lets keep this as is and make a separate PR to fix rules inconsistency?

Reply:
Not sure about this. Once `prettier` will land, we won't have to worry anymore

Reply:
Yeah, that's a problem. Omit the brackets to make the tools happy